wormhole/wormhole_test.go

```diff
@@ -247,46 +253,54 @@ func TestWormholeFileTransportSendRecvViaRelayServer(t *testing.T) {
 
 	url := rs.WebSocketURL()
 
-	relayServer := newTestTCPRelayServer()
-	defer relayServer.close()
+	testDisableLocalListener = true
+	defer func() { testDisableLocalListener = false }()
 
-	var c0 Client
-	c0.RendezvousURL = url
-	c0.TransitRelayURL = "tcp:" + relayServer.addr
+	for relayProtocol, newRelayServer := range relayServerConstructors {
+		t.Run(fmt.Sprintf("With %s relay server", relayProtocol), func(t *testing.T) {
+			relayServer := newRelayServer()
+			relayURL := relayServer.url.String()
+			defer relayServer.close()
 
-	var c1 Client
-	c1.RendezvousURL = url
-	c1.TransitRelayURL = "tcp:" + relayServer.addr
+			var c0 Client
+			c0.RendezvousURL = url
+			c0.TransitRelayURL = relayURL
 
-	fileContent := make([]byte, 1<<16)
-	for i := 0; i < len(fileContent); i++ {
-		fileContent[i] = byte(i)
-	}
+			var c1 Client
+			c1.RendezvousURL = url
+			c1.TransitRelayURL = relayURL
 
-	buf := bytes.NewReader(fileContent)
+			fileContent := make([]byte, 1<<16)
+			for i := 0; i < len(fileContent); i++ {
+				fileContent[i] = byte(i)
+			}
 
-	code, resultCh, err := c0.SendFile(ctx, "file.txt", buf, true)
-	if err != nil {
-		t.Fatal(err)
-	}
+			buf := bytes.NewReader(fileContent)
 
-	receiver, err := c1.Receive(ctx, code, true)
-	if err != nil {
-		t.Fatal(err)
-	}
+			code, resultCh, err := c0.SendFile(ctx, "file.txt", buf)
+			if err != nil {
+				t.Fatal(err)
+			}
 
-	got, err := ioutil.ReadAll(receiver)
-	if err != nil {
-		t.Fatal(err)
-	}
+			receiver, err := c1.Receive(ctx, code)
```

Comment:
`Receive()` now takes 3 parameters. So this would not compile in the current form.

Reply:
Do we want `disableListener` parameter true or false here?

Reply:
You want `disableListener` as false for those tests that just use relay and not transfer from client to client.